docs/project-file-structure.md

```diff
@@ -10,17 +10,34 @@ Please see the [C++ Styleguide](./styleguide-cpp.md) for more details
 
 All godot files are in the OpenVic2/game directory
 
+### Directories
+
+* r/, root/, res/, res://
+    * The root directory of the Godot Project
+* u/, user/, user://
+    * The user directory used by the project (Platform specific)
+    * Windows u/ - %APPDATA%/OpenVic2/
+    * MacOS u/ - ~/Library/Application Support/OpenVic2/
+    * Linux u/ - ~/.local/share/OpenVic2/
+* d/, data/, common/ - The data directory used to store data files
+    * can be a subdirectory of root/ - root/common/
+    * can be a subdirectory of user/ - user/common/
+    * files in this directory should not be included in the godot export, but as external files in the same directory as the executable
+
 ### Definitions
 
-* root/, res/, r/ -> The root directory of the Godot project (OpenVic2/game/)
-* user/, u/ -> The user directory used by the project (Platform specific)
-* core -> related to the functionality of OpenVic2 without modification
+* core - OpenVic2 without modification
+* mod - OpenVic2 with changes in files to modify the gameplay experience
+* scene - godot engine scene object
+* script - godot engine gdscript object
+* resource - godot engine resource object
+* asset - static file generally used as is within the project - art, music etc.
+* data file - static file that holds data or information that needs to be processed to be used - economy good information, province names etc.
+
+### OpenVic2 File Extensions
```

Comment:
We should avoid using our own file extensions when more recognizable extensions already exist and are commonly used.

We can remove lines 37-40

Reply:
There is no decent file extension for our use case when it comes to our own formats, we are not using any `.txt` file extensions, it causes too many problems and its foolish to default to using it when you have a specified format you are parsing, Paradox as a whole was fools to not recognize this fact, we should never duplicate this behavior.